package.json

```
@@ -8,6 +8,7 @@
     "lint": "eslint './*.{ts,js}' '{src,tests}/**/*.{js,ts,vue}' --color",
     "build": "pnpm vite build",
     "build:w": "pnpm vite build --watch --mode development",
```

I think, these should be possible
```suggestion
    "build": "vite build",
    "build:w": "pnpm build --watch --mode development",
```